src/components/Tooltip/Tooltip.tsx

```diff
@@ -171,30 +156,22 @@ const Tooltip: React.FC<TooltipProps> = ({
 
     return (
         <>
-            <TetherComponent
-                {...getAttachmentFromPlacement(placement)}
-                constraints={[
-                    {
-                        to: 'window',
-                        attachment: 'together'
-                    }
-                ]}
-                renderTarget={ref =>
-                    React.cloneElement(children as React.ReactElement, {
-                        onMouseOver: () => handleVisibilityChange(true),
-                        onMouseOut: () => handleVisibilityChange(false),
-                        ref
-                    })
-                }
-                renderElement={(ref: React.RefObject<HTMLDivElement>) =>
-                    isVisible && (
-                        <TooltipBody ref={ref} inverted={inverted}>
-                            {dynamicContent}
-                        </TooltipBody>
-                    )
-                }
-            />
-            <GlobalTetherStyles />
+            {React.cloneElement(children as React.ReactElement, {
+                onMouseOver: () => handleVisibilityChange(true),
+                onMouseOut: () => handleVisibilityChange(false),
+                ref: setTriggerReference
+            })}
+            {isVisible && (
+                <TooltipBody
+                    ref={setContentReference}
+                    inverted={inverted}
+                    style={{ ...styles.popper }}
+                    variant={attributes.popper?.['data-popper-placement']}
```

Comment:
Is this the recommended way of doing this?

Reply:
This is to render the arrow dynamically. Is the only option we found... We saw a library that is called `react-popper-tooltip` and check its implementation and it was using this key to get the actual current placement as the one we get from the props is not reliable because of popper which changes the placement based on the screen size and screen positioning of the tooltip.

About the `popoverTriggerRef` I don't think you're actually using it in the `Popover`, it's being passed to a component but not really used. The used one are the `triggerReference` and `contentReference`. 

> Also what is this variant trying to do with attributes.popper? set up the positioning of the arrow?
Yes

> I wonder if it wouldn't have been easier to just copy the Popover implementation
It's what we did but there are many things that are different and not necessary.